hail/python/hail/vds/combiner/combine.py

```diff
@@ -75,11 +75,10 @@ def make_entry_struct(e, alleles_len, has_non_ref, row):
 
             handled_fields['gvcf_info'] = (hl.case()
                                            .when(hl.is_missing(row.info.END),
-                                                 hl.struct(**(
-                                                     parse_as_fields(
-                                                         row.info.select(*info_to_keep),
-                                                         has_non_ref)
-                                                 )))
+                                                 parse_allele_specific_fields(
```

Comment:
dead-ass didn't realize that `_as_` was an abbreviation until *after* I made this PR.